src/store/index.js

```diff
@@ -16,26 +16,36 @@ export default new Vuex.Store({
     [STATE_KEYS.FILMS]: films,
```

Comment:
#Optional
Since your state is getting a bit bloated, it might be worth looking into Vuex "modules".

Reply:
I don't think there's any reason to divide this store, because all the elements are responsible for the movies.